src/main/java/org/fiximulator/core/FIXimulatorApplication.java

```diff
@@ -572,6 +572,15 @@ public void sendExecution(Execution execution) {
             executionReport.set(new SecurityID(order.getSecurityID()));
             executionReport.set(new IDSource(order.getIdSource()));
         }
+        System.out.println("LiqudityFlag");
```

Comment:
Prefer some sort of 20th century logging mechanism rather than System.out.println.

Reply:
:trollface:

Reply:
@aridmeme please consider implementing the code review tonality recommended in this article: https://medium.com/@ironfroggy/respect-and-code-reviews-83cfd961e1c3

Reply:
Our loggers have been refactored to ingest System.out.println as our core logging mechanism to more easily sync our projects. I may or may not know at all what I am talking about.

Reply:
However I also note that there is a typo in the string literal.

Reply:
👋 @aaronpike @dsprowls-sofi

Reply:
@leeduan HI TO YOU TOO

Reply:
😂😂